allow to compile Slave mode

kgawda opened the pull request:
some corrections in er9x Makefile to fix issues with compiling with SLAVE=YES

MikeBland commented:
I've corrected the caps in the filename on my local copy.
The "extra files" is not really a problem since if you have "SLAVE = YES" you shouldn't have something like "EXT=JETI" as it is meaningless for the slave (used by the 9Xtreme). I have, however, also moved the lines in the makefile as suggested.
I have other changes in the 'next' branch (locally) so that is where I've made these changes.

kgawda commented:
Hi

Just to explain: I have modified the lines regarding EXT because: I have
the FRSKY modification in my Turnigy 9X hardware (and I guess it is
necessary because you otherwise RX/TX is unavailable) and I see that
presence of FRSKY is checked in slave.c code. So I use `make EXT=FRSKY
SLAVE=YES`.

Thanks for taking this into account. Looking forward for seeing the 'next'
branch published :)

Best Regards
Konrad

2016-04-30 22:36 GMT+02:00 MikeBland <notifications@github.com>:

> I've corrected the caps in the filename on my local copy.
> The "extra files" is not really a problem since if you have "SLAVE = YES"
> you shouldn't have something like "EXT=JETI" as it is meaningless for the
> slave (used by the 9Xtreme). I have, however, also moved the lines in the
> makefile as suggested.
> I have other changes in the 'next' branch (locally) so that is where I've
> made these changes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MikeBland/mbtx/pull/19#issuecomment-215993345>
>
